addon/validators/base.js

```diff
@@ -87,23 +87,21 @@ export default Ember.Object.extend({
    */
   createErrorMessage(type, options, value, context) {
     options = options || {};
-    var attribute = options.attributeDescription || 'This field';
```

Comment:
Since you're removing it, you'll need to add the default in your own message hash